src/main/java/io/openliberty/tools/common/plugins/util/ServerFeatureUtil.java

```diff
@@ -108,6 +108,19 @@
      */
     public Set<String> getServerFeatures(File serverDirectory) {
```

Comment:
Looks good, but just a note. I didn't see any calls to this method anymore in the gradle pr. Is this being left in for maven?

Reply:
Yes, left in for maven. I'm going to look at maven next to see if it needs to change also. I'll remove this method when/if maven no longer uses it.